fal/models/season.py

```diff
@@ -25,15 +26,31 @@ class SeasonOfYear(Enum):
 
 @attr.s(auto_attribs=True)
 class Season(OrmFacade):
-    _entity: orm.Season
+    _entity: fal.orm.Season
     season_of_year: SeasonOfYear
     year: int
 
-    min_weeks_between_bench_swaps: int
+    # TODO: move this field to db so we don't have to retrieve it from config file every time
+    min_weeks_between_bench_swaps: int = 3
```

Comment:
Shouldn't we read it from config file for now instead of hardcoding it?

Reply:
yes but patching is such a pain :(

but i will do it

Reply:
oh turns out I don't need to patch